app/components/OutcomeBar.tsx

```diff
@@ -39,6 +40,10 @@ type OutcomeTokenMarginalPricesResponse = {
 
 export const OutcomeBar = ({ market }: OutcomeBarProps) => {
   const { id } = market;
+  const marketModel = new Market(market);
+  const winnerOutcome = marketModel.getWinnerOutcome();
+  const isClosed = marketModel.isClosed;
+  const isClosedAndInvalid = isClosed && !winnerOutcome;
```

Comment:
Why not use `marketModel.isAnswerInvalid` here instead? We are showing grey on finalising and pending markets, although they don't have an answer and are closed, they are not invalid.

Reply:
Here is a visual example

![image](https://github.com/user-attachments/assets/46fba9ec-6cef-48f2-8854-3477fae4c2c5)